spec/spec_helper.rb

```diff
@@ -21,7 +21,9 @@
 backtrace_filter.add_filter(%r{gems/railties})
 backtrace_filter.add_filter(%r{tapioca/helpers/test/})
 
-Minitest::Reporters.use!(SpecReporter.new(color: true), ENV, backtrace_filter)
+unless ENV["RM_INFO"]
+  Minitest::Reporters.use!(SpecReporter.new(color: true), ENV, backtrace_filter)
+end
```

Comment:
Can we make this to be just:
```suggestion
Minitest.backtrace_filter = backtrace_filter
```

Reply:
Can you explain the reasoning? Do we not want to use `SpecReporter`?